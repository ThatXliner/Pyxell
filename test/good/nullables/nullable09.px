
func f<A,B>(A? x) [B] def
    print x
    return [x]

x = null
y = f(x)
print y

[Float?] z = y
print z.map(_ ?? 1.5)
