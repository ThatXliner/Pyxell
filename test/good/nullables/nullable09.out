null
[null]
[1.5]
